python/pyarrow/dataset.py

```diff
@@ -753,7 +753,7 @@ def write_dataset(data, base_dir, basename_template=None, format=None,
                   max_partitions=None, max_open_files=None,
                   max_rows_per_file=None, min_rows_per_group=None,
                   max_rows_per_group=None, file_visitor=None,
-                  existing_data_behavior='error'):
+                  existing_data_behavior='error', create_dir=True):
```

Comment:
Can you add the corresponding parameter description in the docstring below?

Reply:
Added the parameter desc, thanks!